components/form-login-dark.html

```diff
@@ -0,0 +1,31 @@
+<div class="flex items-center justify-center h-screen bg-gray-700 sm:px-6">
+    <div class="w-full max-w-sm p-4 bg-gray-900 rounded-md shadow-md sm:p-6">
+        <div class="flex items-center justify-center">
+            <span class="text-xl font-medium text-white">Login</span>
+        </div>
+        <form class="mt-4">
+            <label for="email" class="block">
+                <span class="text-sm text-white">Email</span>
+                <input type="email" id="email" name="email" autocomplete="username" class="block w-full px-3 py-2 mt-1 text-gray-700 border rounded-md form-input focus:border-indigo-600" required />
+            </label>
+            <label for="password" class="block mt-3">
+                <span class="text-sm text-white">Password</span>
+                <input type="password" id="password" name="password" autocomplete="current-password" class="block w-full px-3 py-2 mt-1 text-gray-700 border rounded-md form-input focus:border-indigo-600" required />
+            </label>
+            <div class="flex items-center justify-between mt-4">
+                <div>
+                    <label class="inline-flex items-center">
+                        <input type="checkbox" class="text-indigo-600 border form-checkbox"/>
+                        <span class="mx-2 text-sm text-white">Remember me</span>
+                    </label>
+                </div>
+                <div>
+                    <a class="block text-sm text-indigo-200 fontme hover:underline" href="#">Forgot your password?</a>
+                </div>
+            </div>
+            <div class="mt-6">
+                <button type="submit" class="w-full px-4 py-2 text-sm text-center text-white bg-indigo-500 rounded-md hover:bg-indigo-500">Sign in</button>
```

Comment:
No hover effect on the button since the background color has been set to the same color.

```suggestion
                <button type="submit" class="w-full px-4 py-2 text-sm text-center text-white bg-indigo-500 rounded-md hover:bg-indigo-700">Sign in</button>
```